#!/bin/usr/python
def raise_exception_msg(message=""):
    raise NameError(message)
